.github/workflows/ui.yml

```diff
@@ -1,33 +0,0 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Any reason to remove it? 

IMO it doesn't hurt to have a minimal sanity test effort even on the ui/ even if we don't use it? I'm looking at it as "best effort" kind of thing.